tests/water.py

```diff
@@ -0,0 +1,20 @@
+import os
+import os.path as osp
```

Comment:
```suggestion
```

Reply:
Not needed here. In modern Python, it's preferable to use `pathlib` for working with the filesystems.